learning-objectives/data.yml

```
@@ -277,22 +277,23 @@ csharp:
   - data-types:
       - primitives
       - non-primitives
+      - structs
+      - enums
       - anonymous-types
       - dynamic-types
   - collections:
       - lists
       - arrays
       - sets
   - variables
-  - control-structures:
-      - conditionals:
-          - if
-          - switch
-      - loops:
-          - for
-          - while
-          - do-while
-          - foreach
+  - conditionals:
+      - if
+      - switch
+  - loops:
+      - for
+      - while
+      - do-while
+      - foreach
```

porque [ahora has simplificado abajo ](https://github.com/Laboratoria/curriculum/pull/1747/files/c7972d1de854b92e1f64b9af4049e5d69aa4ea89..5ba9ef0ed10f4a2ce17cb511e6a15ccc99d226d7#diff-f024bea0816df50add67a51093662e506dbf79d7c66a0c47629428bcbae3bbf4R2178) a `csharp/conditionals` y `csharp/loops` , seria:
```suggestion
  - conditionals
  - loops
```